monai/bundle/workflows.py

```diff
@@ -230,10 +244,11 @@ def __init__(
         tracking: str | dict | None = None,
         workflow_type: str | None = None,
         workflow: str | None = None,
+        properties_path: PathLike | None = None,
         **override: Any,
     ) -> None:
         workflow_type = workflow if workflow is not None else workflow_type
-        super().__init__(workflow_type=workflow_type)
+        super().__init__(workflow_type=workflow_type, properties_path=properties_path)
```

Comment:
Could you please help add 1 unit test to cover this new arg?

Thanks.

Reply:
Hi Nic, added in the latest commit, thanks.